RiotSwiftUI/Modules/Room/UserSuggestion/Coordinator/UserSuggestionCoordinator.swift

```diff
@@ -73,6 +75,11 @@ final class UserSuggestionCoordinator: Coordinator, Presentable {
             
             switch result {
             case .selectedItemWithIdentifier(let identifier):
+                if identifier == "@room" {
```

Comment:
Can we move the `@room` to some shared constant somewhere?

Reply:
Good idea, updated.